docs/installation.md

```diff
@@ -27,74 +27,85 @@ Here is a command to create and activate a new virtual environment with *conda*
-- maintainer: ONLY contains tools for building the documentation and building for PyPI.
+- all: contains mathematical functions, tutorial dependencies and development libraries. 
+- minimal: ONLY contains mathematical functions (so has limited dependencies). 
+- tutorial: ONLY contains mathematical functions and tutorial dependencies. 
+- maintainer: contains tools for building the documentation and building releases. 
+
+Each of the above installation options are available on PyPI. "Minimal" is also available on conda-forge. (In time, we intend to add more installation options to conda-forge.)
 
 ### 1. "All" Dependencies (excludes some maintainer-only packages)
 
 Use this for `scores` development and general use.
 
 Installs:
-* Mathematical API code and libraries
-* Everything needed to run the tutorial notebooks
-* Testing, static analysis and other developer libraries
-* Does **not** install tools for making packages and releasing new versions
+* Mathematical API code and libraries.
+* Everything needed to run the tutorial notebooks.
+* Testing, static analysis and other developer libraries.
+* Does **not** install tools for making packages and releasing new versions.
 
-#### From a Local Checkout of the Git Repository
+#### With pip
 
 ```bash
+# From a local checkout of the Git repository
 pip install -e .[all]
 ```
 
 ### 2. "Minimal" Dependencies (Mathematical API Functions Only)
 Use this to install the `scores` code into another package or system.
 
 Installs:
-* Mathematical API functions and libraries
+* Mathematical API functions and libraries.
 * Only the required core dependencies. Nothing extra - no tutorials, no developer requirements.
-* (Note for high-performance users - dask is not included by default in the minimal install, but will be used if installed into the environment)
+* (Note for high-performance users - dask is not included by default in the minimal install, but will be used if installed into the environment.)
 
 #### From PyPI
 
 ```bash
 pip install scores
 ```
+#### With conda
+
+```bash
+conda install conda-forge::scores
+```
+
+(Note: at present, only the "minimal" installation option is available from conda. In time, we intend to add more installation options to conda.)
 
 ### 3. "Tutorial" Dependencies
 Use this for running tutorials using `scores`, but when you don't need or want developer tools.
 
 Installs:
-* Mathematical API functions and libraries
-* JupyterLab, Plotly, and libraries for reading data, so that the tutorial notebooks can be run
+* Mathematical API functions and libraries.
+* JupyterLab, Plotly, and libraries for reading data, so that the tutorial notebooks can be run.
 
-#### From a Local Git Repository
+#### With pip 
 
 ```bash
+# From a local checkout of the Git repository
 pip install .[tutorial]
 ```
 
 ### 4. "Maintainer" Dependencies
 Use this to build the docs, create packages and prepare new versions for release.
 
```

Comment:
```suggestion
Use this to build the docs, create packages and prepare new versions for release. Note - those wishing to build the documentation locally will need to install `pandoc`. `pandoc` is a complex non-Python package and is typically more reliably installed through the operating system package manager, so is not included in the `scores` installation.
```

Reply:
The wording looks good to me. Do we still need `pandoc` in the pyproject.toml? https://github.com/nci/scores/blob/develop/pyproject.toml#L63

Reply:
The pandoc in pyproject.toml is the "Pandoc Python Library" which is not the same as the command-line pandoc tool which is written in Haskell (i.e. actual pandoc). I'll add a comment to the toml file for people who come across it. It's definitely confusing!

Reply:
Thanks @nicholasloveday for this feedback. @tennlee wants to check a few things before confirming the final wording. As such, he will merge this PR without this change, and come back and put up a new PR to address it.